int/api/html/front/errors.js

```diff
@@ -27,3 +29,24 @@ const errorCodes = new Map([
 function getErrorMessage(errorCode) {
     return errorCodes.get(errorCode);
 }
+
+function handleAPIError(error) {
+    if (error.response) {
+        if (error.response.data) {
+            if (error.response.data.code) {
+                errorAlert(getErrorMessage(error.response.data.code));
+            }
+            console.error("Thyra error:", error.response.data);
+        } else {
+            console.error(error.response);
+        }
+    } else if (error.request) {
+        // The request was made but no response was received
+        console.error(error.request);
+    } else if (error.message) {
+        // Something happened in setting up the request that triggered an Error
+        console.error(error.message);
+    } else {
+        console.error(error);
+    }
+}
```

Comment:
```suggestion
// If the error is from Thyra, we display the error to the user and log the details in the console.
// Otherwise, we simply display the details in the console.
function handleAjaxError(error) {

    if (error.response && error.response.data) {
            if (error.response.data.code) {
                errorAlert(getErrorMessage(error.response.data.code));
            }
            console.error("Thyra error:", error.response.data);
    } else {
        console.error(error);
    }
}
```

Reply:
@SlnPons I'm not sure it's a good idea to don't share with the end user when something went wrong even if the error is not from thyra.

Wdyt?

Reply:
@gregLibert agreed. All errors should be displayed to the end user so that they understand what's going on. 
Just for my understanding currently the errors can come from:
* Thyra 
* Node API (the blockchain)
Do I miss anything?

Reply:
The error can come:
- from the library we use to send request to thyra
- from the browser itself
- from the network
- from the framework we use on thyra side
- from our code on thyra

Only errors from the last case are screened out right now with the proposed code.

Reply:
Ok, thank you. I don't know the best practices about how to manage and list all these errors (same library, separate ones,...). But I know we need to have them listed somewhere AND handled beautifully for us and for the user.

How do you think we should handle them? 
Keeping in mind that the different objects (ie: wallet, storage...) will be removed from Thyra to act as external modules and be used in plug-in.